Sources/TSCBasic/Lock.swift

```diff
@@ -96,12 +96,12 @@ public final class FileLock {
         switch type {
         case .exclusive:
             if !LockFileEx(handle, DWORD(LOCKFILE_EXCLUSIVE_LOCK), 0,
-                           DWORD(INT_MAX), DWORD(INT_MAX), &overlapped) {
+                           UInt32.max, UInt32.max, &overlapped) {
```

Comment:
Should this be `Int32.max` (since I'm assuming that `INT_MAX` is signed), or does the actual value not matter so much?

Reply:
Never mind — I mistakenly thought DWORD was signed.